java/com/fortech/internship/controller/AddressController.java

```diff
@@ -0,0 +1,106 @@
+package com.fortech.internship.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import com.fortech.internship.dao.AddressDAO;
+import com.fortech.internship.dto.AddressDTO;
+import com.fortech.internship.model.Address;
+import com.fortech.internship.service.impl.AddressService;
+
+@RestController
```

Comment:
Just as an idea:
You can put here  @RequestMapping(value="/address") as a general url request mapping for this REST API

Reply:
solved, please check